modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,64 @@
+=== Log Collection
+* With this update, you can deploy multiple, isolated, and RBAC-protected `ClusterLogForwarder` custom resource (CR) instances in any namespace. This allows independent groups to forward desired logs to any destination while isolating their configuration from other collector deployments. (link:https://issues.redhat.com/browse/LOG-1343[LOG-1343])
+
+[IMPORTANT]
+====
+In order to support multi-cluster log forwarding in additional namespaces other than the `openshift-logging` namespace, you must update the Cluster Logging Operator to watch all namespaces. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
+====
+
+* With this update, you can use the flow control or rate limiting mechanism to limit the volume of log data that can be collected or forwarded by dropping excess log records. The input limits prevent badly-behaved containers from overloading the {logging-uc} and the output limits put a ceiling on the rate of logs shipped to a given data store. (link:https://issues.redhat.com/browse/LOG-884[LOG-884])
+
+* With this update, you can configure the log collector to look for HTTP connections and receive logs as an HTTP server, also known as a webhook. (link:https://issues.redhat.com/browse/LOG-4562[LOG-4562])
+
+* With this update, you can configure audit polices to control which Kubernetes and OpenShift API server events are forwarded by the log collector. (link:https://issues.redhat.com/browse/LOG-3982[LOG-3982])
+
+[id="logging-release-notes-5-8-0-log-storage"]
+=== Log Storage
+* With this update, LokiStack administrator can have more fine grained control over who accesses what logs by granting access to logs on a namespace basis. (link:https://issues.redhat.com/browse/LOG-3841[LOG-3841])
+
+* With this update, the Loki Operator introduces `PodDisruptionBudget` configuration on LokiStack deployments to ensure normal operations during {product-title} cluster restarts by keeping ingestion and query path available. (link:https://issues.redhat.com/browse/LOG-3839[LOG-3839])
+
+* With this update, the reliability of existing LokiStack installations are seamlessly improved by applying a set of default Affinity/Anti-Affinity policies.
+(link:https://issues.redhat.com/browse/LOG-3840[LOG-3840])
+
+* With this update, you can manage zone-aware data replication as an administrator in LokiStack, in order to enhance reliability in the event of a zone failure. (link:https://issues.redhat.com/browse/LOG-3266[LOG-3266])
+
+* With this update, a new supported small-scale LokiStack size of 1x.extra-small is introduced for {product-title} clusters hosting a few workloads and smaller ingestion volumes (upto 100GB/day). (link:https://issues.redhat.com/browse/LOG-4329[LOG-4329])
+
+* With this update, the LokiStack administrator has access to an official Loki dashboard to inspect storage performance and health of each component. (link:https://issues.redhat.com/browse/LOG-4327[LOG-4327])
```

Comment:
```suggestion
* With this update, the LokiStack administrator has access to an official Loki dashboard to inspect storage performance and the health of each component. (link:https://issues.redhat.com/browse/LOG-4327[LOG-4327])
```

Reply:
I have changed this to ... to inspect **the** storage performance and **the** health of each component.